app/controllers/concerns/attachment_manageable.rb

```diff
@@ -2,7 +2,24 @@ module AttachmentManageable
   def allow_only_one_attachment
     return unless params[:files].is_a?(Array)
 
-    flash.now[:alert] = t(".attachment_manageable.upload_limit_exceeded")
+    flash.now[:alert] = t("attachment_manageable.upload_limit_exceeded")
     render turbo_stream: turbo_stream.replace("flash", partial: "layouts/shared/flash_messages")
   end
+
+  def show_alert_if_attachment_missing
+    if !params[:pet][:images].nil?
+      no_empty_images = params[:pet][:images].reject { |image| image == "" }
+    elsif !params[:pet][:files].nil?
+      no_empty_files = params[:pet][:files].reject { |image| image == "" }
```

Comment:
I'd also add a comment on why we are rejecting the ""

Reply:
Did this in [chore: add comment on why we remove empty string from files / images …](https://github.com/rubyforgood/homeward-tails/pull/1273/commits/3e3bdb83b7e572b9a2baef81bff2f67f11da91fb)